.github/workflows/config-comment-test.yml

```diff
@@ -119,7 +119,16 @@ jobs:
           fetch-depth: 0
           fetch-tags: true
 
-      - name: Get test type
+      - name: Verify !test
+        run: |
+          if [[ "${{ startsWith(github.event.comment.body, '!test') }}" == "true" ]]; then
+            echo 'Command starts with !test'
+          else
+            echo "::error::Usage: ${{ env.USAGE }}"
+            echo "::error::Command must start with !test to invoke model-config-tests' config-comment-test.yml."
```

Comment:
What happens if someone does `!bump`? Are the compatible?

Reply:
The `!bump` command is handled in https://github.com/ACCESS-NRI/model-config-tests/blob/main/.github/workflows/config-pr-2-confirm.yml, not here. Maybe in future we could consolidate all the comment commands, but that would be a very very large file :D